src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    { person.age !== undefined && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+
+    <p className="Person__partner">
+      {!person.isMarried
+        ? `I am not married`
+        : `${person.partnerName} is my ${getTitle(person)}`}
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable